docs/integrations/ldap/org.md

```diff
@@ -55,15 +55,19 @@ catalog:
             map:
               description: l
             set:
-              metadata.customField: 'hello'
+              - metadata.customField: 'hello'
+                path: 'hello'
+                value: 'hello'
```

Comment:
these last two lines should not be here - it's just a key-value thing and it looks confusing with those present

(same below)

Reply:
Without path and value it doesn't work, I attached the screenshots above, if you mean different thing, i didn't get it

Reply:
Oh oof. The schema is inconsistent with the code.

The schema says `set?: { [key: string]: JsonValue };`. That sounds like the better experience.

The config says `set?: { path: string; value: JsonValue }[];`, and that's what the code does too, indeed. But that doesn't feel particularly smooth.

The reason you had to specify both ways is that one is enforced by the config schema system, and the other one is enforced by the runtime code loading the data. That isn't good at all.

Would you be up to change this so that the code also does `set?: { [key: string]: JsonValue };`? That way, you can delete the bottom two lines here in the docs too

Reply:
Now, documentation and actual code is inconsistent as you know. I can't run without path and value right now. Should I change the config as `set?: { [key: string]: JsonValue };` or what should I do?

Reply:
Yes. The schema is good, the code should be adapted to the schema.